build/actions/update.js

```diff
@@ -10,12 +12,23 @@
     description: 'update the resin cli',
     help: 'Use this command to update the Resin CLI\n\nThis command outputs information about the update process.\nUse `--quiet` to remove that output.\n\nThe Resin CLI checks for updates once per day.\n\nMajor updates require a manual update with this update command,\nwhile minor updates are applied automatically.\n\nExamples:\n\n	$ resin update',
     action: function(params, options, done) {
-      return npm.update(packageJSON.name, function(error, version) {
+      return npm.isUpdated(packageJSON.name, packageJSON.version, function(error, isUpdated) {
         if (error != null) {
           return done(error);
         }
-        console.info("Upgraded " + packageJSON.name + " to v" + version + ".");
-        return done(null, version);
+        if (isUpdated) {
+          return done(new Error('You\'re already running the latest version.'));
+        }
+        return child_process.exec("npm install -g " + packageJSON.name, function(error, stdout, stderr) {
```

Comment:
Line is too long.<br>Identifier 'child_process' is not in camel case.<br>Mixed double and single quotes.